### DISCUSSION TITLE
Is it compatibile to the new postgres 17-alpine?

### DISCUSSION THREAD

**Discussion by update-freak** [TandoorRecipes/recipes#3325] [General]
I saw there was a release of postgres 17-alpine.
Is Tandoor compatible with it?

**Comment by vabene1111** [comment]
likely, altough tandoors django version is a bit older at the moment so maybe lookup django 4.2 compatibility with PG 17

**Comment by update-freak** [comment]
I tried now the PG17 with the current release. Works fine till now :)
Is the PG recommended instead of SQLite because of better performace?

**Reply by vabene1111** [reply]
yes and because of some database specific features that are lacking in sqlite.
